### PR TITLE
Document System.MidpointRounding enum

### PR DESCRIPTION
**Pull request by carlossanlop**
@tannergooding is the area owner.

I obtained the descriptions from the API proposal: https://github.com/dotnet/corefx/issues/31902
Here's the PR that introduced them: https://github.com/dotnet/coreclr/pull/20815/files

### CODE REVIEW

File: xml/System/MidpointRounding.xml [hunk 1]
```diff
@@ -187,7 +187,7 @@
       </ReturnValue>
       <MemberValue>3</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The result closest to and no greater than the infinitely precise result.</summary>
```
Comment (tannergooding) [inline xml/System/MidpointRounding.xml]: These should likely be prefixed with `When a number is halfway between two others, it is rounded toward`

Comment (carlossanlop) [reply xml/System/MidpointRounding.xml]: Like this?:

```suggestion
        <summary>When a number is halfway between two others, it is rounded toward
 the result closest to and no greater than the infinitely precise result.</summary>
```

File: xml/System/MidpointRounding.xml [hunk 2]
```diff
@@ -219,7 +219,7 @@
       </ReturnValue>
       <MemberValue>4</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The result closest to and no less than the infinitely precise result.</summary>
```
Comment (carlossanlop) [inline xml/System/MidpointRounding.xml]:
```suggestion
        <summary>When a number is halfway between two others, it is rounded toward
 the result closest to and no less than the infinitely precise result.</summary>
```

File: xml/System/MidpointRounding.xml [hunk 3]
```diff
@@ -251,7 +251,7 @@
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The result closest to and no greater in magnitude than the infinitely precise result.</summary>
```
Comment (carlossanlop) [inline xml/System/MidpointRounding.xml]:
```suggestion
        <summary>When a number is halfway between two others, it is rounded toward the result closest to and no greater in magnitude than the infinitely precise result.</summary>
```

**Comment by carlossanlop** [comment]
@rpetrusha @tannergooding I apllied Tanner's suggestion.

**Comment by carlossanlop** [comment]
This is fully cooked, @rpetrusha, @tannergooding . Can we get it merged if you don't have any additional suggestions?

**Comment by rpetrusha** [comment]
Yes, I'll merge it now, @carlossanlop.

**Pull request merged**